examples/FirebaseUIChat/FirebaseUIChat.xcodeproj/project.pbxproj

```diff
@@ -7,19 +7,19 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		C887CB2F78EAA58DC354FAFB /* libPods-FirebaseUIChat.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 585EBE168DE871301F5FA598 /* libPods-FirebaseUIChat.a */; };
```

Comment:
Please run `pod deintegrate` to get rid of the CoocaPods artifacts in the project.